conf/modules.config

```diff
@@ -1127,34 +1118,35 @@ process {
 //
 
 process {
+    withName: '.*ANALYSE_MT:MERGE_ANNOTATE_MT:.*' {
+        ext.when = { params.mt_fasta_shift && params.mt_intervals && !(params.analysis_type == "wes") }
+    }
 
     withName: '.*ANALYSE_MT:MERGE_ANNOTATE_MT:GATK4_MERGEVCFS_LIFT_UNLIFT_MT' {
         ext.prefix = { "${meta.id}_merged" }
-        ext.when = { params.mt_fasta_shift && params.mt_intervals && !(params.analysis_type == "wes") }
     }
 
     withName: '.*ANALYSE_MT:MERGE_ANNOTATE_MT:GATK4_VARIANTFILTRATION_MT' {
         ext.prefix = { "${meta.id}_filt" }
-        ext.when = { params.mt_fasta_shift && params.mt_intervals && !(params.analysis_type == "wes") }
     }
 
     withName: '.*ANALYSE_MT:MERGE_ANNOTATE_MT:SPLIT_MULTIALLELICS_MT' {
         ext.args = '--output-type z --multiallelics -both'
         ext.prefix = { "${meta.id}_split" }
-        ext.when = { params.mt_fasta_shift && params.mt_intervals && !(params.analysis_type == "wes") }
         publishDir = [
             enabled: false
         ]
     }
 
     withName: '.*ANALYSE_MT:MERGE_ANNOTATE_MT:TABIX_TABIX_MT' {
-        ext.when = { params.mt_fasta_shift && params.mt_intervals && !(params.analysis_type == "wes") }
+        publishDir = [
+            enabled: false
+        ]
```

Comment:
is the publishDir directive necessary or is it just that you need something here in order for the switch to work?

Reply:
My idea is to have an entry for all the processes so this is sort of like explicitly stating the defaults. They don't have to be there

Reply:
I like it!